.github/workflows/codesee-arch-diagram.yml

```diff
@@ -0,0 +1,23 @@
+# This workflow was added by CodeSee. Learn more at https://codesee.io/
```

Comment:
I don't think we can use this action for a number of reasons, one of which being vendor validation, which is an entire process, that entails a request, a justification, a security check, and more.

Can we skip this for now?